tests/common.go

```diff
@@ -2852,10 +2857,10 @@ func CreateCluster(name string, kubeconfigPath string, orgID string, cloud_name
 }
 
 // createS3BackupLocation creates backup location
-func createS3BackupLocation(name string, uid, cloudCred string, cloudCredUID, bucketName string, orgID string, encryptionKey string) {
-	Step(fmt.Sprintf("Create S3 backup location [%s] in org [%s]", name, orgID), func() {
-		CreateS3BackupLocation(name, uid, cloudCred, cloudCredUID, bucketName, orgID, encryptionKey)
-	})
+func createS3BackupLocation(name string, uid, cloudCred string, cloudCredUID, bucketName string, orgID string, encryptionKey string) error {
+	log.InfoD("Create S3 backup location [%s] in org [%s]", name, orgID)
+	err := CreateS3BackupLocation(name, uid, cloudCred, cloudCredUID, bucketName, orgID, encryptionKey)
+	return err
```

Comment:
If you don't want to return verbose error you can just do
```
return CreateS3BackupLocation(name, uid, cloudCred, cloudCredUID, bucketName, orgID, encryptionKey)
```

Reply:
done